doc/pmp.rst

```diff
@@ -0,0 +1,33 @@
+.. _pmp:
+
+Physical Memory Protection (PMP)
+================================
+
+The Physical Memory Protection (PMP) unit implements region-based memory access checking in-accordance with the RISC-V Privileged Specification, version 1.11.
+The following configuration parameters are available to control PMP checking:
+
++----------------+---------------+-------------------------------------------------+
+| Parameter      | Default value | Description                                     |
++================+===============+=================================================+
+| PMPEnable      | 0             | PMP checking enabled                            |
```

Comment:
Same here: `PMP support enabled` instead of `PMP checking enabled`.

Reply:
done